docs/flutter/v2/guides/quickstart.mdx

```diff
@@ -5,316 +5,868 @@ nav: 2.0
+
+```dart
+// Called when there's a track update - use to update local & remote track variables
+  @override
+  void onTrackUpdate(
+      {required HMSTrack track,
+      required HMSTrackUpdate trackUpdate,
+      required HMSPeer peer}) {
+    if (track.kind == HMSTrackKind.kHMSTrackKindVideo) {
+      switch (trackUpdate) {
+        case HMSTrackUpdate.trackRemoved:
+          if (mounted) {
+            setState(() {
+              peer.isLocal
+                  ? localPeerVideoTrack = null
+                  : remotePeerVideoTrack = null;
+            });
+          }
+          return;
+        default:
+          if (mounted) {
+            setState(() {
+              peer.isLocal
+                  ? localPeerVideoTrack = track as HMSVideoTrack
+                  : remotePeerVideoTrack = track as HMSVideoTrack;
+            });
+          }
+      }
+    }
+  }
 ```
 
-### Leave a video call
+### Step 6: Other callbacks
+
+100ms SDK provides various other callbacks to handle different scenarios in the app. For example, you can use `onAudioDeviceChanged` to get updates whenever a new audio device or an audio device is switched. Please check [here](./../features/update-listeners) for more information about these callbacks. 
+
+```dart
+// More callbacks - no need to implement for quickstart
+  // More callbacks - no need to implement for quickstart
+  @override
+  void onAudioDeviceChanged(
+      {HMSAudioDevice? currentAudioDevice,
+      List<HMSAudioDevice>? availableAudioDevice}) {}
+
+  @override
+  void onChangeTrackStateRequest(
+      {required HMSTrackChangeRequest hmsTrackChangeRequest}) {}
+
+  @override
+  void onHMSError({required HMSException error}) {}
+
+  @override
+  void onMessage({required HMSMessage message}) {}
+
+  @override
+  void onReconnected() {}
 
-To leave a video call, just call leave on HMSSDK.
+  @override
+  void onReconnecting() {}
 
-```dart section=LeaveVideoCall sectionIndex=1
-hmsSDK.leave();
+  @override
+  void onRemovedFromRoom(
+      {required HMSPeerRemovedFromPeer hmsPeerRemovedFromPeer}) {}
+
+  @override
+  void onRoleChangeRequest({required HMSRoleChangeRequest roleChangeRequest}) {}
+
+  @override
+  void onRoomUpdate({required HMSRoom room, required HMSRoomUpdate update}) {}
+
+  @override
+  void onUpdateSpeakers({required List<HMSSpeaker> updateSpeakers}) {}
 ```
 
-## Glossary
+### Step 7: Render video in a tile
+
+We had initialized the `HMSUpdateListener` class in the [Implement meeting page section](#step-3-implement-meeting-page); now, we can use the same to render video tracks in a tile.
+
+To display a video track, first get the `HMSVideoTrack` & pass it on to `HMSVideoView` using the `setVideoTrack` function. Ensure to attach the `HMSVideoView` to your UI hierarchy. Check the [render video guide](./../features/render-video) for more information. 
```

Comment:
`using the setVideoTrack function.` should be removed.

Reply:
done